packages/chain-adapters/src/ChainAdapterManager.ts

```diff
@@ -38,7 +38,23 @@ export class ChainAdapterManager {
             const ws = new unchained.bitcoin.ws.Client(wsUrl)
             return this.addChain(
               type,
-              () => new bitcoin.ChainAdapter({ providers: { http, ws }, coinName: 'Bitcoin' })
+              () => new bitcoin.ChainAdapter({ providers: { http, ws }, coinName: 'Cosmos' })
+            )
+          }
+
+          case ChainTypes.Cosmos: {
+            const http = new unchained.cosmos.api.V1Api(
+              new unchained.cosmos.api.Configuration({ basePath: httpUrl })
+            )
+            return this.addChain(
+              type,
+              () =>
+                new cosmos.ChainAdapter({
+                  providers: {
+                    http
+                  },
+                  symbol: 'ATOM'
+                })
```

Comment:
Seems like weird linting? Also, bitcoin uses verbiage `coinName` vs cosmos is using verbiage `symbol`. Might be nice to standardize?